JavaScript/Github_Good_First_Issues/DB.js

```diff
@@ -0,0 +1,9 @@
+let repoDB = [
+    {
+        "owner": "HarshCasper",
+        "name": "Rotten-Scripts"
+
```

Comment:
Can you add another project to this: https://github.com/TesseractCoding/NeoAlgo 

Just as an example :wink:

Reply:
Sure 😁